packages/workflow/src/ErrorReporterProxy.ts

```diff
@@ -0,0 +1,33 @@
+export interface IReportingOptions {
+	tags: Record<string, string>;
+	extra: Record<string, unknown>;
+}
+
+interface IErrorReporter {
+	error: (error: Error, options?: IReportingOptions) => void;
+	warn: (warning: string, options?: IReportingOptions) => void;
+}
+
+class ErrorReporterProxy {
+	constructor(readonly reporter: IErrorReporter) {}
+}
+
+const isTesting = process.env.NODE_ENV === 'test';
```

Comment:
Having test-specific logic outside of tests is not ideal. Can we use the logger? It logs based on `N8N_LOG_LEVEL` and this is what silences logs in tests.

Reply:
should I change this to `NODE_ENV !== 'production'` instead ? that way we can avoid error reports from development as well.